README.md

```diff
@@ -9,26 +9,27 @@ A simplified API to Intel(R) oneAPI Data Analytics Library that allows for fast
 - [Documentation](https://intelpython.github.io/daal4py/)
 - [scikit-learn API and patching](https://intelpython.github.io/daal4py/sklearn.html#sklearn)
 - [Source Code](https://github.com/IntelPython/daal4py/tree/master/src)
+- [Building from Sources](INSTALL.md)
```

Comment:
That file contains more than just building from sources, I suggest you change the alt text for this link.
You can also add this link at the end of Installation section with some explanation of what INSTALL.md contains.

Reply:
It contains building daal4py from sources + building documentation from sources only. So, I suppose the name covers this. Do you see anything else in INSTALL.md?

Reply:
You are right. I looked through the file and the hierarchy confused me, it made it seem that the file covers different things. Sorry.

Reply:
Anyway, I added different heading in INSTALL.md and added a link in "Installation". Thank you.